### PR TITLE
Added Days parameter to AWS costs

### PR DESCRIPTION
**Pull request by jdesouza**
This PR fixes #

## Checklist
* [x] I have signed the CLA
* [x] I have updated/added any relevant documentation

### CODE REVIEW

File: plugins/aws-costs/version.txt [hunk 1]
```diff
@@ -1 +1 @@
-1.1.3
+1.1.4
```
Comment (rbren) [inline plugins/aws-costs/version.txt:1]: maybe we make this a minor version bump--WDYT?

**Review by ivanfetch-fw — APPROVED** [review]

File: plugins/aws-costs/aws-costs.sh [hunk 2]
```diff
@@ -62,12 +67,18 @@ while [ ! $# -eq 0 ]; do
...
+if [[ "$days" = "" ]]; then
+  days='5'
+fi
+if [[ "$tagkey" = "" || "$tagvalue" = "" || "$database" = "" || "$table" = "" || "$catalog" = "" || "$workgroup" = "" || "$days" = "" ]]; then
   usage
   exit 1
 fi
 
-initial_date_time=$(date -u -d '5 day ago' +"%Y-%m-%d %H:00:00.000")
+initial_date_time=$(date -u -d  $days+' day ago' +"%Y-%m-%d %H:00:00.000")
```
Comment (vitorvezani) [inline plugins/aws-costs/aws-costs.sh:81]: It seems like this will yield `5' day ago'` instead of `'5 day ago'`

Comment (jdesouza) [reply plugins/aws-costs/aws-costs.sh:81]: I think it's fine:
james@DESKTOP-A3283C9:~$ export days=5
james@DESKTOP-A3283C9:~$ initial_date_time=$(date -u -d  $days+' day ago' +"%Y-%m-%d %H:00:00.000")
james@DESKTOP-A3283C9:~$ echo $initial_date_time
2023-02-18 13:00:00.000
james@DESKTOP-A3283C9:~$

Comment (ivanfetch-fw) [reply plugins/aws-costs/aws-costs.sh:81]: The string calculates the relative date via the date command - it returns output like: 2023-02-18 13:00:00.000

**Review by vitorvezani — APPROVED** [review]

**Pull request merged**